### PR TITLE
[J9] consume jdt version in maven/gradle that does support Java 9

### PR DESCRIPTION
**Pull request by cdietrich**
[J9] consume jdt version in maven/gradle that does support Java 9
https://github.com/eclipse/xtext-extras/issues/211

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java [hunk 1]
```diff
@@ -113,6 +113,17 @@ public boolean apply(Issue issue) {
 
 	private static Logger log = Logger.getLogger(XtendBatchCompiler.class.getName());
 
+	private static final boolean isJava9CompilerApiAvailable;
+	static {
+		boolean result = true;
+		try {
+			Class.forName("javax.lang.model.element.ModuleElement");
+		} catch (ClassNotFoundException e) {
```
Comment (kthoms) [inline org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java]: you could assign `isJava9CompilerApiAvailable=true` in the try clause and remove the result variable

Comment (cdietrich) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java]: code will be deleted as soon as oxygen2 is available

Comment (kthoms) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java]: so just wait until tomorrow?

Comment (cdietrich) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java]: or january, dont know when theyll publish to maven

Comment (kthoms) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java]: so temporary commit to make it work meanwhile? maybe if thursday JDT is not available on central?

Comment (cdietrich) [reply org.eclipse.xtend.core/src/org/eclipse/xtend/core/compiler/batch/XtendBatchCompiler.java]: no ill wait

File: gradle/versions.gradle [hunk 2]
```diff
@@ -8,8 +8,9 @@ ext.versions = [
 	'xtext': version,
 	'xtext_gradle_plugin': '1.0.20',
 	'idea' : project.hasProperty('ideaEAP') ? 'LATEST-EAP-SNAPSHOT' : '145.972.3',
-	'jdtCore': '3.12.2',
-	'jdtCompilerApt': '1.2.100',
+	'jdtCore': '3.13.100',
+	'jdtCompilerApt': '1.3.60',
+	'jdtCompilerTool': '1.2.51',
 	'emfMwe2': '2.9.1.201705291010',
```
Comment (szarnekow) [inline gradle/versions.gradle]: Should we use a more recent mwe2 version

Comment (cdietrich) [reply gradle/versions.gradle]: there is no newer mwe yet

**Pull request merged**